tests/test_band_masks.py

```diff
@@ -44,28 +44,28 @@ def tiffs(tmpdir):
 
 def test_mask_flags():
     with rasterio.open('tests/data/RGB.byte.tif') as src:
-        for flags in src.mask_flags:
-            assert flags & MaskFlags.nodata
-            assert not flags & MaskFlags.per_dataset
-            assert not flags & MaskFlags.alpha
```

Comment:
sounds good, I don't think I've come across anything that uses `mask_flags` external to Rasterio